src/playground/sample_data_tooling/README.md

```diff
@@ -0,0 +1,124 @@
+
+In order to use default arguments, the tooling requires three environment variables to be defined:
+- `SAMPLE_DATA_HOST`
+- `SAMPLE_DATA_USERNAME`
+- `SAMPLE_DATA_PASSWORD`
+
+To use this tool locally, set `SAMPLE_DATA_HOST` to `localhost`. Then, follow this sequence of commands:
+```
+kubectl get pods
+```
+
+Then, copy one of the OpenSearch pods. For this example, we will use `opensearch-cluster-leader-0`:
+```console
+$ kubectl get pods
+NAME                                               READY   STATUS      RESTARTS   AGE
+dashboards-opensearch-dashboards-cd954cd4b-7hdsk   1/1     Running     0          10d
+dashboards-opensearch-dashboards-cd954cd4b-sp74n   1/1     Running     0          10d
+opensearch-cluster-leader-0                        1/1     Running     0          10d
+opensearch-cluster-leader-1                        1/1     Running     0          10d
+opensearch-cluster-leader-2                        1/1     Running     0          10d
+opensearch-cluster-leader-3                        1/1     Running     0          10d
+opensearch-cluster-leader-4                        1/1     Running     0          10d
+```
+
+Finally, `port-forward` this opensearch pod to port `9200`.
+```
+kubectl port-forward opensearch-cluster-leader-0 9200
+```
+
+To use this tooling, from this directory, navigate to `sample_data_jobs` and run
+```
+python3 startup_job.yaml
+
+python3 refresh_job.yaml
+```
+
+To run the startup job and refresh job, respectively. See `sample_data_jobs/README.md` for details over arguments.
+
+## Deploying tooling as a container
+
+In order to deploy this tooling to Playground, first ensure that [`kubectl`](https://kubernetes.io/docs/tasks/tools/) is installed. If you are not using Playground but instead another [Kubernetes cluster](https://kubernetes.io/docs/setup/), install [Helm](https://helm.sh/docs/intro/install/) and then install [OpenSearch using Helm](https://opensearch.org/docs/latest/opensearch/install/helm/) as well as [OpenSearch Dashboards using Helm](https://opensearch.org/docs/latest/dashboards/install/helm/).
+
+Once `kubectl` is installed and the Kubernetes cluster is running, first configure secrets so the container can pull credentials from the cluster.
+
+```
+kubectl create secret generic sample-data-secret --from-literal='username=<admin username>' --from-literal='password=<admin password>' --from-literal='host=opensearch-cluster-leader'
+```
+
+Then, from this directory, change directory into the job and cronjob deployment files and deploy them.
+
+For the sample data startup job:
+```
+cd ../../../config/playground/job
+
+kubectl apply -f startup_deployment.yaml
+```
+
+This job should start immediately once it has been deployed.
+
+To remove this job, call:
+```
+kubectl delete job sample-data-startup-job
+```
+
+For the sample data refresh cronjob:
+```
+cd ../../../config/playground/cronjob
```

Comment:
can we change the folder structure here? sample_data_tooling/cronjob?

Reply:
Done 👍🏽